test/foundry/unit/StakingRewardsV2/StakingRewardsV2.t.sol

```diff
@@ -1041,33 +1009,12 @@ contract StakingRewardsV2Test is DefaultStakingV2Setup {
         assertEq(initialEscrowTokenBalance, finalEscrowTokenBalance);
     }
 
-    function test_unstakeEscrowSkipCooldown_Does_Not_Change_Token_Balances() public {
```

Comment:
**Though:** Can we have an equivalent of this test with `unstakeEscrowAdmin`?

Reply:
Added `test_unstakeEscrowAdmin_Does_Not_Change_Token_Balances`